src/bundle/Resources/config/services.yml

```
@@ -1,9 +1,17 @@
 parameters:
     ezplatform.cron.registry.cronjobs.class: EzSystems\EzPlatformCronBundle\Registry\CronJobsRegistry
+    ezplatform.console.command.run.class: EzSystems\EzPlatformCronBundle\Command\CronRunCommand
 
 services:
     ezplatform.cron.registry.cronjobs:
         class: '%ezplatform.cron.registry.cronjobs.class%'
         arguments:
             - '%kernel.environment%'
             - '@ezpublish.siteaccess'
+
+    ezplatform.console.command.run:
+        class: '%ezplatform.console.command.run.class%'
+        arguments:
+            $logger: "@monolog.logger.cronjob"
```

CS: inconsistent quotes.
```suggestion
            $logger: '@monolog.logger.cronjob'
```

---

@alongosz The two latest issues are fixed with the new commit